sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -105,6 +105,9 @@ export class EducationProgramApi extends HttpBaseClient {
   /**
    * Gets location programs option list authorized for students.
    * @param locationId location id.
+   * @param programYearId program year id.
+   * @param isIncludeInActiveProgramYear isIncludeInActiveProgramYear, if isIncludeInActiveProgramYear, then both active
+   * and not active program year is considered.
```

Comment:
I did a test to view application with inactive program and noticed an issue. Please take care of it.

![image](https://github.com/bcgov/SIMS/assets/54600590/6a543c69-6803-4191-b8c5-cee4bb342e95)

Reply:
Please also test PIR in read only view. 

It uses a common drop down.